paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTest.kt

```diff
@@ -47,50 +55,64 @@ internal class PaymentOptionsViewModelTest {
     private val prefsRepository = FakePrefsRepository()
     private val customerRepository = FakeCustomerRepository()
     private val paymentMethodRepository = FakeCustomerRepository(PAYMENT_METHOD_REPOSITORY_PARAMS)
-    private val lpmResourceRepository = StaticLpmResourceRepository(mock())
+    private val lpmRepository = mock<LpmRepository>().apply {
```

Comment:
Why did this become necessary? And if we really need it, can we use an actual implementation of `LpmRepository` instead of a mock?